### PR TITLE
Update join,start and end chat messages

### PR DESCRIPTION
**Pull request by Pablete1234**
More chat message updates (tested code)

### CODE REVIEW

File: src/main/java/in/twizmwaz/cardinal/module/modules/match/MatchModule.java [hunk 1]
```diff
@@ -31,24 +31,16 @@ public void unload() {
...
         if (event.getTeam() != null) {
             if (event.getTeam().size() == 1) {
-                ChatUtils.getGlobalChannel().sendLocalizedMessage(new UnlocalizedChatMessage(ChatColor.DARK_PURPLE + "# # {0}" + ChatColor.DARK_PURPLE + " # #", new LocalizedChatMessage(ChatConstant.UI_MATCH_WIN, event.getTeam().getColor() + ((Player) event.getTeam().get(0)).getName())));
+                ChatUtils.getGlobalChannel().sendLocalizedMessage(new UnlocalizedChatMessage("{0}" , new LocalizedChatMessage(ChatConstant.UI_MATCH_WIN, event.getTeam().getColor() + ((Player) event.getTeam().get(0)).getName() + ChatColor.WHITE)));
             } else {
-                ChatUtils.getGlobalChannel().sendLocalizedMessage(new UnlocalizedChatMessage(ChatColor.DARK_PURPLE + "# # {0}" + ChatColor.DARK_PURPLE + " # #", new LocalizedChatMessage(ChatConstant.UI_MATCH_WIN, event.getTeam().getCompleteName())));
+                ChatUtils.getGlobalChannel().sendLocalizedMessage(new UnlocalizedChatMessage("{0}" , new LocalizedChatMessage(ChatConstant.UI_MATCH_WIN, event.getTeam().getCompleteName() + ChatColor.WHITE )));
```
Comment (ryanw-se) [inline src/main/java/in/twizmwaz/cardinal/module/modules/match/MatchModule.java]: Why do you include ```ChatColor.WHITE``` at the end of these messages?

**Comment by Pablete1234** [comment]
Opinions?

**Comment by iEli2tyree011** [comment]
Perhaps we should not reduce the appearance of the game over message until we have a util for sending titles to players. Everything else seems okay, though.

**Comment by Pablete1234** [comment]
Isn't that a sportbukkit/bukkit feature itself? do you really need an util to be able to use titles?

**Comment by CaptainElliott** [comment]
@Pablete1234 it's a utility feature, making it easier and cleaner to code

**Comment by dentmaged** [comment]
@CaptainElliott I'll happily do it.

**Comment by Pablete1234** [comment]
I updated the messages for who touched a monument to remove colors from "and" and "," (gray to white), and "others" and "the enemy" (dark_aqua to white)

**Pull request closed without merging**